frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -211,23 +184,41 @@ export default {
         }
       })
     },
-    instance_create_update: function(audiosurfer_id, start_time, end_time) {
```

Comment:
Split `instance_create_update` into separate `instance_create` and `instance_update` methods. This is to satisfy single responsibility principal and also to fix the bug. Because a child ( `waveform_selector`) component fires `create` event when a new annotation region rendered, during initial rendering of saved annotation regions `instance_create_update` was getting fired. Because these annotations were already stored by `audio_annotation_core` component using `this.instance_list`, this action was processed as an update command propagated to the `annotation_ui_factory` and `audio_toolbar` component. `audio_toolbar` componnet would then enable saving since the change was detected. To sum it up, initial rendering of the annotated auido regions had a side effect of enabling "Save" button.